ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OMOJOModelTestSuite.scala

```diff
@@ -463,6 +463,70 @@ class H2OMOJOModelTestSuite extends FunSuite with SharedH2OTestContext with Matc
     }
   }
 
+  test("getStartType is exposed for loaded model") {
+    val mojo = H2OMOJOModel.createFromMojo(
+      this.getClass.getClassLoader.getResourceAsStream("multi_model_iris.mojo"),
+      "multi_model_iris.mojo")
+
+    mojo.getStartTime() shouldBe 1631392711317L
+  }
+
+  test("getStartType is exposed for freshly trained model") {
+    val (_, mojo) = multinomialModelFixture()
+
+    mojo.getStartTime() should not be 0L
```

Comment:
I would go with `mojo.getStartTime() should be > 0L`

Reply:
Slightly academic point of view, but this would increase chance of accidental test failure - e.g. in case the system clock is set before 1.1.1970 for whatever reason.

Reply:
We can also just save the timestamp before we start made the mojo and compare to this timestamp.

-1 sometimes (not in this specific case) indicates null/NA in h2o, it is a good suggestion by @krasinski